### PR TITLE
Make default not savething online

### PR DESCRIPTION
**Pull request by BartLucien**
This PR will correct the following things:
* Make it impossible for guest user to save playlist online
* Remove old playlist if the playlist name is changed in edition.
* Do not show save online popup when editing playlist
* Add the link to our video in the ReadMe

### CODE REVIEW

**Review by sim0mo — APPROVED** [review]
It's okay

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -160,4 +160,6 @@
     <string name="deleteUndoQuestion">Annuler la suppression?</string>
     <string name="offline_error_message_toast">Impossible d\'acceder à des ressources en ligne</string>
     <string name="no_lyrics_generation_offline">Impossible de générer des paroles</string>
+    <string name="guest_cannot_save_online">Il faut être connecté in avec google pour sauver une playlist en ligne</string>
```
Comment (VictorSchneuwly) [inline app/src/main/res/values-fr/strings.xml:163]: Typo: You can remove the "in" and add a capital letter to Google

**Review by VictorSchneuwly — CHANGES_REQUESTED** [review]
A couple of typos

**Comment by VictorSchneuwly** [comment]
Corrected the typos myself since @BartLucien had other obligations

**Pull request merged**